upgrade/sql/9.0.0.sql

```diff
@@ -451,3 +451,11 @@ INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VAL
 /* Reorganize search aliases */
 /* https://github.com/PrestaShop/PrestaShop/pull/37470 */
 /* PHP:ps_900_reorganize_aliases_tab(); */
```

Comment:
Just trying something

```suggestion
/* PHP:ps_900_reorganize_aliases_tab(); */;
```